src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -1,20 +1,32 @@
 package seedu.address.model.transaction;
 
+import java.util.Date;
+import java.util.Objects;
+import java.util.Set;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
```

Comment:
From Travis with love : 
"Wrong lexicographical order for 'java.time.Clock' import. Should be before 'java.util.Set'.
Wrong lexicographical order for 'java.time.Instant' import. Should be before 'java.util.Set'. 
Wrong lexicographical order for 'java.time.ZoneId' import. Should be before 'java.util.Set'. "